src/plugins/intel_cpu/src/transformations/snippets/x64/pass/eliminate_brgemm_copy_b.cpp

```diff
@@ -36,20 +43,33 @@ pass::EliminateBrgemmCopyB::EliminateBrgemmCopyB() {
             return false;
         }
 
+        // Update external repacking config for the further pipeline stages
```

Comment:
Please move this comment to a proper place: we don't update the config here. Maybe move it on L51 instead of "`// Mark this input as repacked`"?

Reply:
Updated